modules/backup-s3/client.go

```diff
@@ -179,17 +225,30 @@ func (s *s3Client) WriteToFile(ctx context.Context, backupID, key, destPath stri
 	return nil
 }
 
-func (s *s3Client) Write(ctx context.Context, backupID, key string, r io.ReadCloser) (int64, error) {
+func (s *s3Client) Write(ctx context.Context, backupID, key, overrideBucket, overridePath string, r io.ReadCloser) (int64, error) {
 	defer r.Close()
-	path := s.makeObjectName(backupID, key)
+	client, err := s.getClient(ctx)
+	if err != nil {
+		return -1, errors.Wrap(err, " write: cannot get client")
```

Comment:
let's remove the leading space from the error message

Reply:
done